Immediate crash after external repo import and then directly after start

I've found this while checking #251, it seems to be vector drawable related.
Newest Android Emulator version running with SDK 16 (like my real device) and Android 4.1.2, app version is my patch-4 branch, logcat is:
```
    03-25 12:38:20.022 2597-2597/com.manichord.mgit E/AndroidRuntime: FATAL EXCEPTION: main
                                                                  android.content.res.Resources$NotFoundException: File res/drawable/ic_default_author.xml from drawable resource ID #0x7f020072. If the resource you are trying to use is a vector resource, you may be referencing it in an unsupported way. See AppCompatDelegate.setCompatVectorFromResourcesEnabled() for more info.
                                                                      at android.content.res.Resources.loadDrawable(Resources.java:1923)
                                                                      at android.content.res.Resources.getDrawable(Resources.java:659)
                                                                      at com.nostra13.universalimageloader.core.DisplayImageOptions.getImageForEmptyUri(DisplayImageOptions.java:138)
                                                                      at com.nostra13.universalimageloader.core.ImageLoader.displayImage(ImageLoader.java:251)
                                                                      at com.nostra13.universalimageloader.core.ImageLoader.displayImage(ImageLoader.java:209)
                                                                      at com.nostra13.universalimageloader.core.ImageLoader.displayImage(ImageLoader.java:316)
                                                                      at me.sheimi.android.utils.BasicFunctions.setAvatarImage(BasicFunctions.java:48)
                                                                      at me.sheimi.sgit.adapters.RepoListAdapter.bindView(RepoListAdapter.java:149)
                                                                      at me.sheimi.sgit.adapters.RepoListAdapter.getView(RepoListAdapter.java:97)
                                                                      at android.widget.AbsListView.obtainView(AbsListView.java:2271)
                                                                      at android.widget.ListView.measureHeightOfChildren(ListView.java:1244)
                                                                      at android.widget.ListView.onMeasure(ListView.java:1156)
                                                                      at android.view.View.measure(View.java:15172)
                                                                      at android.widget.RelativeLayout.measureChildHorizontal(RelativeLayout.java:617)
                                                                      at android.widget.RelativeLayout.onMeasure(RelativeLayout.java:399)
                                                                      at android.view.View.measure(View.java:15172)
                                                                      at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:4816)
                                                                      at android.widget.FrameLayout.onMeasure(FrameLayout.java:310)
                                                                      at android.support.v7.widget.ContentFrameLayout.onMeasure(ContentFrameLayout.java:139)
                                                                      at android.view.View.measure(View.java:15172)
                                                                      at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:4816)
                                                                      at android.support.v7.widget.ActionBarOverlayLayout.onMeasure(ActionBarOverlayLayout.java:393)
                                                                      at android.view.View.measure(View.java:15172)
                                                                      at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:4816)
                                                                      at android.widget.FrameLayout.onMeasure(FrameLayout.java:310)
                                                                      at android.view.View.measure(View.java:15172)
                                                                      at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:4816)
                                                                      at android.widget.LinearLayout.measureChildBeforeLayout(LinearLayout.java:1390)
                                                                      at android.widget.LinearLayout.measureVertical(LinearLayout.java:681)
                                                                      at android.widget.LinearLayout.onMeasure(LinearLayout.java:574)
                                                                      at android.view.View.measure(View.java:15172)
                                                                      at android.view.ViewGroup.measureChildWithMargins(ViewGroup.java:4816)
                                                                      at android.widget.FrameLayout.onMeasure(FrameLayout.java:310)
                                                                      at com.android.internal.policy.impl.PhoneWindow$DecorView.onMeasure(PhoneWindow.java:2148)
                                                                      at android.view.View.measure(View.java:15172)
                                                                      at android.view.ViewRootImpl.performMeasure(ViewRootImpl.java:1850)
                                                                      at android.view.ViewRootImpl.measureHierarchy(ViewRootImpl.java:1102)
                                                                      at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:1275)
                                                                      at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1000)
                                                                      at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:4214)
                                                                      at android.view.Choreographer$CallbackRecord.run(Choreographer.java:725)
                                                                      at android.view.Choreographer.doCallbacks(Choreographer.java:555)
                                                                      at android.view.Choreographer.doFrame(Choreographer.java:525)
                                                                      at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:711)
                                                                      at android.os.Handler.handleCallback(Handler.java:615)
                                                                      at android.os.Handler.dispatchMessage(Handler.java:92)
                                                                      at android.os.Looper.loop(Looper.java:137)
                                                                      at android.app.ActivityThread.main(ActivityThread.java:4745)
                                                                      at java.lang.reflect.Method.invokeNative(Native Method)
                                                                      at java.lang.reflect.Method.invoke(Method.java:511)
                                                                      at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:786)
                                                                      at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:553)
                                                                      at dalvik.system.NativeStart.main(Native Method)
                                                                   Caused by: org.xmlpull.v1.XmlPullParserException: Binary XML file line #1: invalid 
    03-25 12:38:22.312 2597-2597/com.manichord.mgit I/Process: Sending signal. PID: 2597 SIG: 9
```

--- a/app/src/main/java/me/sheimi/android/activities/SheimiFragmentActivity.java
+++ b/app/src/main/java/me/sheimi/android/activities/SheimiFragmentActivity.java
@@ -4,6 +4,7 @@ import android.app.AlertDialog;
 import android.content.DialogInterface;
 import android.content.Intent;
 import android.os.Bundle;
+import android.support.graphics.drawable.VectorDrawableCompat;
 import android.support.v7.app.AppCompatActivity;
 import android.view.KeyEvent;
 import android.view.LayoutInflater;
@@ -261,8 +262,8 @@ public class SheimiFragmentActivity extends AppCompatActivity {
         DisplayImageOptions mDisplayOptions = new DisplayImageOptions.Builder()
                 .cacheInMemory(true)
                 .cacheOnDisk(true)
-                .showImageForEmptyUri(R.drawable.ic_default_author)
-                .showImageOnFail(R.drawable.ic_default_author)
+                .showImageForEmptyUri(VectorDrawableCompat.create(getResources(), R.drawable.ic_default_author, getTheme()))
+                .showImageOnFail(VectorDrawableCompat.create(getResources(), R.drawable.ic_default_author, getTheme()))
                 .build();
         File cacheDir = StorageUtils.getCacheDirectory(this);
         ImageLoaderConfiguration configuration = new ImageLoaderConfiguration.Builder(this)

